pkg/kubectl/main.go

```diff
@@ -17,6 +18,13 @@ import (
 
 func Main() {
 	kubenv := os.Getenv("KUBECONFIG")
+	for i, arg := range os.Args {
+		if strings.Contains(arg, "--kubeconfig=") {
+			kubenv = strings.Split(arg, "=")[1]
+		} else if strings.Contains(arg, "--kubeconfig") && i+1 < len(os.Args) {
```

Comment:
Could this be "equal to"?

Reply:
Having `i+1 <= len(os.Args)` would result in a panic if you just give a blank `--kubeconfig`, I know because I had that on the first pass :)

Reply:
I think he meant equal on the string.Contains, not the Args len.

Reply:
I was talking arg len in case there's nothing passed afterwards.